jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/JibExtension.java

```diff
@@ -220,8 +177,7 @@ public ContainerParameters getContainer() {
     return container;
   }
 
-  // TODO: Make @Internal (deprecated)
-  @Input
+  @Internal
```

Comment:
Hmm, should we just get rid of these instead?

Reply:
I suppose.